### ISSUE TITLE
CLI tool is not uploading any XMP files

### ISSUE THREAD

**Issue by ericswpark** [immich-app/immich#9891]
### The bug

The CLI tool will not upload any sidecar files. Tested with CLI tool released with v1.105.1

### The OS that Immich Server is running on

unRAID

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

N/A

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
N/A
```

### Your .env content

```Shell
N/A
```

### Reproduction steps

```bash
1. Upload photos with the CLI tool

The XMP file naming does not seem to matter (i.e. it will not upload `IMG_0001.png.xmp` nor `IMG_0001.xmp`. Capitalization also does not matter (i.e. it will not upload `IMG_0001.XMP`).
```

**Comment by ericswpark** [comment]
@mplawner that sounds like it should be filed as a separate issue as it would indicate Immich doesn't support XMP at all despite having it in the docs. I'm curious as to whether it worked previously but was mistakenly broken during feature addition or if it only works when the assets are uploaded via a certain path (maybe the Android app). Because right now uploading via the CLI tool doesn't work, and the iOS app doesn't have support for reading sidecar metadata as the underlying Flutter library can't access it IIRC.

**Comment by jrasm91** [comment]
This was likely fixed in #9831
